### DISCUSSION TITLE
A question on formulas and in distribution model

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#743] [Questions & Problems]
Hi, all,

When I learn the formula in MoBi, I find this passive transport **PassiveDiffusionPl2Int** is defined as "dN/dt=fu * P_endothelial * SA * (C_pls - C_int / K_int_pls)"; isn't the term "C_pls - C_int / K_int_pls" is equal to 0? 
![image](https://user-images.githubusercontent.com/54141158/66763498-81d4d100-eeda-11e9-8ab2-6c5d623f2eb6.png)

**Comment by HenrikCordes** [comment]
Dear @wangwei1619,

passive diffusion will only take place, if there is a difference in unbound concentration between the two spaces (plasma and interstitial). 
In MoBI you can select "create process rate parameter" and "plot parameter" to have a look at the PassiveDiffusionPl2Int in one of your simulations. You will find that PassiveDiffusionPl2Int is basically never 0 (except for the turning point or when there is no drug in the system any more).

On the other hand, if you set all clearane parameter (hepatic, enzymatic & renal) in your PBPK model to 0 and simulate your system for a long time periode (1000 h or more) you will find that at some point PassiveDiffusionPl2Int will become 0. This is when the system is in equilibrium and the unbound concentration in bothe spaces (here, plasma and interstitial) are equal. 

Hope this helps!

Kind regards,
Henrik

**Comment by wangwei1619** [comment]
Dear Henrik,

Thanks for your advice. But I don't know how to create process rate parameter and to plot it. When I right-clicked on the simulation, I didn't find "create process rate parameter" .
![image](https://user-images.githubusercontent.com/54141158/66834058-8c9d6d80-ef8f-11e9-8a66-25598137d118.png)
Now, I get your idea that unbound concentration between the two spaces (plasma and interstitial) are not equal during equilibrium process, but, another problem comes. I thought Cu + Cb = Ctotal in a compart because V *(Cu + Cb)= V * Ctotal. However, as the PK-Sim takes account protein as another compart, I think unbound and bound drug don't share the same distribution volume, is it right? And, basing on this change, what is the relationship among Cu, Cb, C_water, C_protein, Vf(water/plasma) and Vf(protein/plasma)?

Best regards,
Wang Wei

**Comment by HenrikCordes** [comment]
Dear @wangwei1619,

in MoBi you have to open the "passive transport" building block:
![image](https://user-images.githubusercontent.com/4817133/66849255-e0e82380-ef76-11e9-8713-561e987ff24c.png)
and than go to PassiveDiffusionInt2Cell and check the boxes:
![image](https://user-images.githubusercontent.com/4817133/66849423-24429200-ef77-11e9-85e4-0ca6dd66474f.png)

Kind regards,
Henrik

**Comment by wangwei1619** [comment]
Hi, Henrik,
Thanks for your help, I got it. But I am still confusing with these parameters Cu, Cb, C_water, C_protein, Vf(water) and Vf(protein). According to the calculation method of PK-Sim standard method:
![image](https://user-images.githubusercontent.com/54141158/66920328-a0130c00-f055-11e9-99d3-cf320521406c.png)
it can deduce that C_pls*fu=Cu=C_water=Nu*V_water_pls, which means fu here is equal to (Nu*V_pls)/(N_pls*V_water_pls). It is not identical to what is generally used that fu=Cu/(Cu+Cb)=Nu/(Nu+Nb)=Nu/N_pls. On the other hand, Cu could be assumed to equal C_water, because plasma almost consists of water.
